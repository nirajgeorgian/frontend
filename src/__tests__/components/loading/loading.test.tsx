import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import PageLoading from 'components/loading/page'

test('loading of loading component', () => {
	const { container, getByText } = render(<PageLoading />)
	console.log(container)
	expect(getByText(/loading/)).toBeInTheDocument()
	expect(container).toMatchSnapshot()
})
