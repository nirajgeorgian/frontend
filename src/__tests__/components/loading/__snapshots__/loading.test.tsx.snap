// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading of loading component 1`] = `
<div>
  <div>
    <p>
      loading ...
    </p>
  </div>
</div>
`;
